server/internal/app/file.go

```diff
@@ -0,0 +1,46 @@
+package app
+
+import (
+	"io"
+	"mime"
+	"net/http"
+	"path"
+
+	"github.com/labstack/echo/v4"
+	"github.com/reearth/reearth-cms/server/internal/usecase/gateway"
+)
+
+func serveFiles(
+	ec *echo.Echo,
+	repo gateway.File,
+) {
+	if repo == nil {
+		return
+	}
+
+	fileHandler := func(handler func(echo.Context) (io.Reader, string, error)) echo.HandlerFunc {
+		return func(ctx echo.Context) error {
+			reader, filename, err := handler(ctx)
+			if err != nil {
+				return err
+			}
+			ct := "application/octet-stream"
+			if ext := path.Ext(filename); ext != "" {
+				ct2 := mime.TypeByExtension(ext)
+				if ct2 != "" {
+					ct = ct2
+				}
+			}
+			return ctx.Stream(http.StatusOK, ct, reader)
+		}
+	}
+
+	ec.GET(
+		"/assets/:filename",
```

Comment:
```suggestion
		"/assets/:uuid/:filename",
```

- As with GCS, UUID should be required to access assets.
- Support files in directories such as 3D Tiles

e.g.

```
GET /assets/xxxx-xxxx-xxxx-xxxx/hogehoge.zip
GET /assets/xxxx-xxxx-xxxx-xxxx/hogehoge/tileset.json
```

Reply:
I thought that it should be like this:
GET /assets/xx/xxxx-xxxx-xxxx-xxxx/hogehoge/tileset.json

Reply:
OK `/assets/xx/xxx-xxxx-xxxx/hogehoge.zip`

Reply:
Done!